### DISCUSSION TITLE
possible to queue individual shares ?

### DISCUSSION THREAD

**Discussion by ihubgit** [Jacalz/rymdport#106] [Q&A] [answered]
Hi TBH haven't tested this out yet, but wanted to have an answer before considering doing so.
If i split up a large file into multiple segments and drop them all in the share window, when i share all the individual codes to the recipient, can he start the first part and will the rest automatically follow or must he manually validate each segment once the earlier one finished downloading?

**Comment by Jacalz** [comment]
If you drop more than one file (on upcoming v3.5.0) it will bundle them together into a single directory send. All files must be in the same parent folder and the send will get the same name as the parent folder.

**Comment by ihubgit** [comment]
so if i understood correctly, it won't generate an individual code for each file ?
Is this the default behavior for individual files that are located in the same parent folder ?
Reason I'm asking is in case of problem with internet connection, what happens in this user case ?
Does the recipient have to start all over again ?
Advantage of segments is that at least a part of them will have been transferred.

**Reply by Jacalz** [reply]
> so if i understood correctly, it won't generate an individual code for each file ?

That is correct. You are dropping multiple files at one single time and I think it makes sense to treat that as meaning to send all those files in one go. If you drop files individually, you will get separate codes.

> Is this the default behavior for individual files that are located in the same parent folder ?

You can only drop multiple files at once that are in the same parent folder (error otherwise), but yes, it is the default and currently only behaviour.

> Reason I'm asking is in case of problem with internet connection, what happens in this user case ?
Does the recipient have to start all over again ?
Advantage of segments is that at least a part of them will have been transferred.

I'm bit sure how the wormhole protocol handles network problems. If the network connection drops entirely, I would guess that you need to start over.

**Comment by ihubgit** [comment]
ok thanks for clearing that up.
So in order to mitigate the danger to have to start all over again in case of Internet failure, best way to proceed would be to individually drop the segments and share their individual codes.
In such a user case, how does the app work if i share all the codes to the recipient ?
The recipient can only enter one code at a time right ?
That would mean that he has to manually add each new code for each new sements manually after the former share finished is that correct ?
If so, would there be a way to queue the different shares and give the ability to the recipient to enter multiple codes so that the download of the next segment happens automatically after the completion of the former segment ?
That would be a **huge** timesaver really.

**Reply by Jacalz** [reply, marked as answer]
Download the application and try sending multiple files. You can send to the same computer if you just wish to test stuff. Most of what you are asking you can easily find out yourself.

There is no need to wait for the previous transfer to complete before you start a new one. Just enter the code and press the button and repeat as many times as you have codes.

**Comment by ihubgit** [comment]
ok thanks, i just wanted to be sure before doing a batch transfer with a newbie recipient as me !
